packages/google_maps_flutter/google_maps_flutter_ios/example/ios13/ios/RunnerUITests/RunnerUITests.m

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+static const CGFloat kWaitTime = 15;
+
+@interface RunnerUITests : XCTestCase
+
+@end
+
+@implementation RunnerUITests
+
+- (void)testDraggableCallbacks {
+  XCUIApplication *application = [[XCUIApplication alloc] init];
+  [application launch];
+  XCUIElement *placeMarkerButton = application.staticTexts[@"Place marker"];
+  if (![placeMarkerButton waitForExistenceWithTimeout:kWaitTime]) {
+    NSLog(@"application.debugDescription: %@", application.debugDescription);
```

Comment:
Doesn't it dump the tree to console automatically if an element can't be found?

Reply:
No it does not, we need to manually log it unfortunately. 
It does print out some sort of "tree" but not the entire tree, for example:

```
Failed to tap "Add1" Button: No matches found for Elements matching predicate '"Add1" IN identifiers' from input {(
    Button, label: 'Add',
    Button, label: 'Remove', Disabled,
    Button, label: 'change info', Disabled,
    Button, label: 'change info anchor', Disabled,
    Button, label: 'change alpha', Disabled,
    Button, label: 'change anchor', Disabled,
    Button, label: 'toggle draggable', Disabled,
    Button, label: 'toggle flat', Disabled,
    Button, label: 'change position', Disabled,
    Button, label: 'change rotation', Disabled,
    Button, label: 'toggle visible', Disabled,
    Button, label: 'change zIndex', Disabled,
    Button, label: 'set marker icon', Disabled,
    Button, label: 'Back',
    Button, identifier: 'my_location', label: 'My location',
    Button, label: 'Google Maps'
)}
```